server/routes/topic_page/html.py

```diff
@@ -96,6 +96,11 @@ def topic_page(topic_id=None, place_dcid=None):
 
   more_places = request.args.getlist('places')
 
+  # TODO: should use place metadata API to fetch these data in one call.
+  place_type = place_api.get_place_type(place_dcid)
```

Comment:
Can just use dc.get_place_info() to fetch metadata?

Reply:
done